Makefile

```diff
@@ -23,6 +23,13 @@ gofmt:
 ui:
 	cd ui && npm install && npm run bower install && npm run grunt build
 
+.PHONY: release
+release: image
+	git tag -a $(TAG) -m $(TAG)
+	git push origin $(TAG)
```

Comment:
Yep, still trying to determine the best way to configure the goreleaser tool to create a release on the same repo that we tag. Otherwise we'll tag `aidenkeating/mcp-standalone` but still release on `feedhenry/mcp-standalone`.

Only obvious way is through changing the YAML config directly. Hoping there's some way to override config through the command line. Have asked on their chat so will make that change once I find the best way to go about it.